resources/assets/js/Modal/ModalAddFeedback.js

```diff
@@ -0,0 +1,46 @@
+import React, {Component} from 'react';
+import axios from 'axios';
+import Field from '../Form/Field';
+
+export default class ModalAddFeedback extends Component {
+  constructor(props) {
+    super();
+    this.state = {
+      submission_id: props.submissionID,
+      grade: null,
+      feedback: ""
+    }
+  }
+
+  submit() {
+    const formData = new FormData();
+    formData.append('grade', this.state.grade);
+    formData.append('feedback', this.state.feedback);
+    formData.append('submission_id', this.state.submission_id);
+    axios.post('/submissions/feedback', formData, {
+      headers: {
+        'content-type': 'multipart/form-data'
+      }
+    }).then((response) => {
+      if (response.data.status == 1) {
+        this.props.onCancel();
+      }
+    });
+  }
```

Comment:
You only need to use `formData` and `multipart/form-data` when uploading files.
For basic POST params, you can put the params inline. Also as you are sending through your entire state, you might as well do this to clean things up:

`axios.post('/submissions/feedback', this.state).then(response => {
  if (response.data.status == 1) {
    this.props.onCancel();
  }
});